src/openms/source/ANALYSIS/ID/FalseDiscoveryRate.cpp

```diff
@@ -485,17 +486,17 @@ namespace OpenMS
       it->setHigherScoreBetter(false);
       const vector<ProteinHit>& old_hits = it->getHits();
       vector<ProteinHit> new_hits;
-      for (auto hit : old_hits)
+      for (auto hit : old_hits) // NOTE: performs copy
```

Comment:
I think here the copy is intended since we dont want to change old_hits? @cbielow